src/constants.py

```diff
@@ -1,4 +1,33 @@
+ADD_LABELS = "add_labels"
+REMOVE_LABELS = "remove_labels"
+
 BLOCK_MERGE_VERIFY_CONTEXT = "Verified label"
+
 LABEL_VERIFIED = "Verified"
 LABEL_APPROVE = "Approve"
 NEEDS_MAINTAINERS_APPROVE = "Needs Maintainers Approve"
+
+COMMAND = "command"
+COMMAND_MAJOR = "major"
+COMMAND_MINOR = "minor"
+COMMAND_PATCH = "patch"
+
+BLOCK_MERGE_SEMVER_CONTEXT = "Semver label"
+LABEL_SEMVER_MAJOR = f"{COMMAND_MAJOR.capitalize()}/Breaking-Change"
+LABEL_SEMVER_MINOR = f"{COMMAND_MINOR.capitalize()}/Non-Breaking-Feature"
+LABEL_SEMVER_PATCH = f"{COMMAND_PATCH.capitalize()}/Non-Breaking-Fix"
+
+COMMAND_LABELS_MAP = {
+    COMMAND_MAJOR: {
+        ADD_LABELS: [LABEL_SEMVER_MAJOR],
+        REMOVE_LABELS: [LABEL_SEMVER_MINOR, LABEL_SEMVER_PATCH],
+    },
+    COMMAND_MINOR: {
+        ADD_LABELS: [LABEL_SEMVER_MINOR],
+        REMOVE_LABELS: [LABEL_SEMVER_MAJOR, LABEL_SEMVER_PATCH],
+    },
+    COMMAND_PATCH: {
+        ADD_LABELS: [LABEL_SEMVER_PATCH],
+        REMOVE_LABELS: [LABEL_SEMVER_MAJOR, LABEL_SEMVER_MINOR],
+    },
+}
```

Comment:
To get labels to remove, you can do
```python
labels = [LABEL_SEMVER_MAJOR, LABEL_SEMVER_MINOR, LABEL_SEMVER_PATCH]
list(filter(lambda x: label.capitalize() not in  x, labels))
```

In [20]: list(filter(lambda x: 'major'.capitalize() not in  x, labels))
Out[20]: ['Minor/Non-Breaking-Feature', 'Patch/Non-Breaking-Fix']